src/sbvr-api/permissions.ts

```diff
@@ -387,12 +391,56 @@ const $checkPermissions = (
 	});
 };
 
-const constrainedPermissionError = new PermissionError();
-const generateConstrainedAbstractSql = (
+const convertToLamda = (filter: AnyObject, identifier: string) => {
+	const replaceObject = (object: AnyObject) => {
```

Comment:
What is this function for?

Reply:
I'll add a comment to it as well.
Basically we need to inject all occurences of properties for the new lambda function
for example if there is an `or` operator, where different properties of
the target resource are used, we need to change all occurences of these.
We do this in a recursive way to also cover all nested cases like `or [ and, and ]`

Reply:
Ah, so this handles adding the lambda alias where previously it didn't exist, that makes sense